README.md

```diff
@@ -36,6 +36,10 @@ The following example attempts to outline most of the key features of the DSL.
 # The couchdb database from which to request the changes feed
 changes "http://user:pass@host:port/invoicing" do
 
+  # upsert_mode       # Optional flag which can be enabled to take advantage of Postgres 9.5's support for INSERT CONFLICT,
+  #                   # e.g. upserts.
+  # ember_pouch_mode  # Optional flag which can be enabled if ember-pouch is being used to populate the CouchDB database.
```

Comment:
Better name suggestion by @afinne - `relational_pouch_mode`. Will fix and re-test.